src/Makefile.am

```diff
@@ -637,17 +637,13 @@ clean-local:
 	  $(CPPFLAGS) $(AM_CXXFLAGS) $(AM_CXXFLAGS) $(PIE_FLAGS) $(CXXFLAGS) -c -o $@ $<
 
 check-symbols: $(bin_PROGRAMS) $(bin_SCRIPTS)
-if GLIBC_BACK_COMPAT
 	@echo "Checking glibc back compat of [$(bin_PROGRAMS) $(bin_SCRIPTS)]..."
 	$(AM_V_at) READELF=$(READELF) CPPFILT=$(CPPFILT) $(top_srcdir)/contrib/devtools/symbol-check.py $(bin_PROGRAMS) $(bin_SCRIPTS)
-endif
 
 check-security: $(bin_PROGRAMS) $(bin_SCRIPTS)
-if HARDEN
 	@echo "Checking binary security of [$(bin_PROGRAMS) $(bin_SCRIPTS)]..."
 	$(AM_V_at) READELF=$(READELF) OBJDUMP=$(OBJDUMP) $(top_srcdir)/contrib/devtools/security-check.py $(bin_PROGRAMS)
 	$(AM_V_at) READELF=$(READELF) OBJDUMP=$(OBJDUMP) $(top_srcdir)/contrib/devtools/security-check.py --allow-no-canary $(bin_SCRIPTS)
-endif
```

Comment:
This shouldn't be removed, because it alters the API of `make check-security`, which previously was to check that the binary security matched what the build was configured for. Instead, add an `else` case that makes explicit that binary security checks are skipped if `HARDEN` is disabled.